packages/engine/src/interaction/components/InteractableComponent.ts

```diff
@@ -60,6 +68,33 @@ export enum XRUIActivationType {
   proximity = 0,
   hover = 1
 }
+
+/**
+ * Adds an interactable UI to the entity if it has label text
+ * @param entity
+ */
+const addInteractableUI = (entity: Entity) => {
+  if (!isClient || getMutableState(EngineState).isEditing.value) return //no xrui in editor
```

Comment:
unnecessary checks

Reply:
can you elaborate? I figured we don't want to add an XRUI on the server, and we don't want to run this while editing

Reply:
nevermind, i think i see what you mean after removing the empty dependency useEffect. The remaining useEffect onMount already checked for editing.

Theorizing that isClient checks are redundant for xrUI anyway?